doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -322,6 +322,22 @@ In addition to "appliances", there may also be "facilities".  Again, under the h
 
 ### User interface
 
+#### Bring back points pool in chargen: there is a reason we moved away from it
+
+The main reason would be that it was not a fun system. To quote Venera3:
+
+> The point system turning every character into a Heavy Sleeper Truth Teller Wool Allergy Stimulant Psychosis \<and now insert the traits you want to take\> is just shit design
+
+The point system did not encourage players to try new approaches or roleplay but instead supported attempts to 'break' it, find a way to trick the system, and try to find as much profit within existing points as possible. Which brings us to issue number two, which is...
+
+Points are a bad tool for balance. How many points is a bad back trait worth? 3? 2? 2.5? 2.76? π? Numbers have difficulties representing the abstract influence of a single trait on your entire playthrough. 
+
+Luckily for us, a smart person coded a smarter solution to this, named Survivor mode, which is, put simply, "let's put the character against multiple in-game simulations and see how it would behave". Instead of assuming "dense bones = two points", we apply dense bones mutation to the test character and calculate how many blows it can take against a zombie. Is it more than average? It will help you in your game, so be "strong". Such evaluation will ensure the system will always work without much maintenance burden.
```

Comment:
> let's

Should be "lets", as in "allows" or "enables".

Reply:
it's a contraction, "let us"